src/offsetPath.js

```diff
@@ -18,12 +18,33 @@
       return undefined;
     } else if (ray !== null) {
       inputType = 'ray';
-      var rayInput = ray[1];
-      var rayInputDegrees = parseAngleAsDegrees(rayInput);
-      if (rayInputDegrees === null) {
-        return undefined;
+      var rayInput = ray[1].split(/\s+/);
+      var result = {type: inputType,
+                    input: 'none',
+                    contain: false,
+                    size: 'none'
+                   };
+
+      for (var i = 0; i < rayInput.length; i++) {
```

Comment:
This looks like it will accept ray(0deg contain contain farthest-side closest-side 50deg 90deg).